### ISSUE TITLE
Consider rewriting style guide in rst format with tables

### ISSUE THREAD

**Issue by awalterschulze** [WhatsApp/erlfmt#80]
https://gist.github.com/awalterschulze/75b84d81f85c2db2bf343036ee1dfebb

**Comment by michalmuskala** [comment]
I like this idea. We would need to amend the verification code, though.

Is there a way to stretch the table to take up the whole width with each column taking 50% of the space? That would be ideal.

**Comment by awalterschulze** [comment]
Yes changing the verification code is not going to be as simple as with markdown.

RST seems to have several ways to increase width, but none of them work for me :(

**Comment by awalterschulze** [comment]
It is going to be too hard to parse the rst tables, I hope it is okay to settle for the classic markdown.
